src/core/operationRepo/OperationRepo.ts

```diff
@@ -7,6 +7,8 @@ import { logMethodCall } from '../../shared/utils/utils';
 export class OperationRepo {
   public executorStore: ExecutorStore;
   private _unsubscribeFromModelRepo: () => void;
+  private _deltaQueue: CoreDelta<SupportedModel>[] = [];
+  static DELTAS_BATCH_PROCESSING_TIME = 1;
```

Comment:
Consider re-using
```
import { DELTA_QUEUE_TIME_ADVANCE } from '../../support/constants';
```

Reply:
I don't think we should be using it here because I think there should be a one way dependency between files in `__test__` to `src` - and `DELTA_QUEUE_TIME_ADVANCE` is in `__test__/support/constants.ts `